Doc/whatsnew/3.11.rst

```diff
@@ -181,6 +181,12 @@ Other CPython Implementation Changes
   hash-based pyc files now use ``siphash13``, too.
   (Contributed by Inada Naoki in :issue:`29410`.)
 
+* When an active exception is re-raised by a :keyword:`raise` statement with no parameters,
+  the traceback attached to this exception is now always ``sys.exc_info()[1].__traceback__``.
```

Comment:
The use of ``sys.exc_info()[1]`` always puts me off -- it's just too cryptic. Maybe "the traceback attached to this exception is now always its ``__traceback__`` attribute? Or use the phrasing from the reference manual above?